src/app/store/Cart/Cart.dispatcher.js

```diff
@@ -77,6 +77,7 @@ export class CartDispatcher {
     addProductToCart(dispatch, options) {
         const { product, quantity } = options;
         const { item_id, quantity: originalQuantity } = this._getProductInCart(product);
+        console.log('_getProductInCart', this._getProductInCart(product));
```

Comment:
Please, get rid of Console.log usage and from inappropriate logics from your commit.